test/test_verify.py

```diff
@@ -501,6 +509,14 @@ def runTestSet(driver, testSet, nameSalt, enable_stress_test):
         testAutoTableCreation = TestAutoTableCreation(driver, nameSalt, schemaRegistryAddress, testSet)
         testAutoTableCreationTopic2Table = TestAutoTableCreationTopic2Table(driver, nameSalt, schemaRegistryAddress, testSet)
 
+        testSchemaEvolutionJson = TestSchemaEvolutionJson(driver, nameSalt)
+        testSchemaEvolutionAvroSR = TestSchemaEvolutionAvroSR(driver, nameSalt)
+
+        testSchemaEvolutionWithAutoTableCreationJson = TestSchemaEvolutionWithAutoTableCreationJson(driver, nameSalt)
+        testSchemaEvolutionWithAutoTableCreationAvroSR = TestSchemaEvolutionWithAutoTableCreationAvroSR(driver, nameSalt)
+
+        testSchemaEvolutionNonNullableJson = TestSchemaEvolutionNonNullableJson(driver, nameSalt)
+
```

Comment:
great tests thanks for adding them. 

should we also add a test where we have multiple partitions and each partitions gets different schema?

Reply:
we do have the test, see TestSchemaEvolutionJson, it contains two topics with different schema but going to the same table